great_expectations/self_check/util.py

```diff
@@ -99,14 +100,14 @@
     from pyspark.sql import SparkSession
     from pyspark.sql.types import StructType
 except ImportError:
-    SparkDataFrame = type(None)  # type: ignore
-    SparkSession = None  # type: ignore
-    StructType = None  # type: ignore
+    SparkDataFrame = type(None)
```

Comment:
@Kilo59 How come some values are `None` and others `type(None)` -- what should our decision process be?  Thanks!

Reply:
Good question, I didn't add these so I don't know what the thinking was here.

Reply:
@Kilo59 The reason I asked is that they show up with the plus and in green here in this file-based review. :)

Reply:
The only thing I did was remove the `type: ignore` comments